<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setupDialog</class>
 <widget class="QDialog" name="setupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>341</width>
     <height>301</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="portBox"/>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>BaudRate:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="baudRateBox"/>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>DataBits:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="dataBitsBox"/>
   </widget>
   <widget class="QSplitter" name="splitter_4">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>StopBits:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="stopBitsBox"/>
   </widget>
   <widget class="QSplitter" name="splitter_5">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Parity:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="parityBox"/>
   </widget>
   <widget class="QSplitter" name="splitter_6">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Flow Control:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="flowControlBox"/>
   </widget>
   <widget class="QSplitter" name="splitter_7">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>QueryMode:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="queryModeBox"/>
   </widget>
   <widget class="QSplitter" name="splitter_8">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Timeout:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="timeoutBox">
     <property name="suffix">
      <string> ms</string>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="text">
     <string>Terminal Mode</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
  <tabstop>queryModeBox</tabstop>
  <tabstop>portBox</tabstop>
  <tabstop>parityBox</tabstop>
  <tabstop>stopBitsBox</tabstop>
  <tabstop>dataBitsBox</tabstop>
  <tabstop>timeoutBox</tabstop>
  <tabstop>flowControlBox</tabstop>
  <tabstop>baudRateBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>setupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>setupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
